packages/mux-uploader/src/index.ts

```diff
@@ -337,8 +365,7 @@ class MuxUploaderElement extends HTMLElement {
 
   setProgress(percent: number) {
     if (this.uploadPercentage) this.uploadPercentage.innerHTML = `${Math.floor(percent)}%`;
-    this.setAttribute('aria-valuenow', `${Math.floor(percent)}`);
-    this.setAttribute('aria-valuetext', `${Math.floor(percent)} percent`);
+    this.progressBar?.setAttribute('aria-valuenow', `${Math.floor(percent)}`);
```

Comment:
non-blocking: why remove `aria-valuetext`?

Reply:
Cool I think that's a good enough reason to prefer `aria-valuenow` for now. We can always revisit from a UX perspective which vocalizations would be preferred by users (where these tradeoffs bottom out imo, esp if the dev overhead is minimal)